Lib/multiprocessing/popen_spawn_win32.py

```diff
@@ -84,6 +96,10 @@ def wait(self, timeout=None):
                     code = -signal.SIGTERM
                 self.returncode = code
 
+                # bpo-33929: Don't trigger the finalizer yet since
```

Comment:
I'm not sure this comment is useful.

Reply:
The first version of my patch tried to close sentinel as soon as the process completed. But it broke a test. Then I understood that it's public attribute, and so it cannot be closed here. For me it's non-obvious that close() cannot be closed "soon" (at least, not by Popen itself).

Reply:
I don't know. The comment confuses me more than the situation it's supposed to comment :-)